configuration/external_parameters.rst

```diff
@@ -80,24 +40,86 @@ You can now reference these parameters wherever you need them.
 
         <doctrine:config>
             <doctrine:dbal
-                driver="pdo_mysql"
-                dbname="symfony_project"
-                user="%database.user%"
-                password="%database.password%"
+                host="%env(DATABASE_HOST)%"
             />
         </doctrine:config>
 
     .. code-block:: php
 
         $container->loadFromExtension('doctrine', array(
             'dbal' => array(
-                'driver'   => 'pdo_mysql',
-                'dbname'   => 'symfony_project',
-                'user'     => '%database.user%',
-                'password' => '%database.password%',
+                'host' => '%env(DATABASE_HOST)%',
             )
         ));
 
+You can also give the ``env(...)`` parameters a default value: the default value
+will be used whenever the corresponding environment variable is *not* found:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        #app/config/parameters.yml
+
+        parameters:
+            env(DATABASE_HOST): localhost
```

Comment:
What's documented here is:
```yaml
parameters:
    database_driver: '%env(DATABASE_DRIVER)%'
    env(DATABASE_DRIVER): pdo_mysql
```

Which, as you can see, is quite different from the format you tested. If there is something we can do to improve the description of the feature, please tell us!